platform/linux-generic/odp_classification.c

```diff
@@ -190,6 +190,8 @@ int odp_cls_capability(odp_cls_capability_t *capability)
 	capability->supported_terms.bit.tcp_sport = 1;
```

Comment:
Commit subject is too long. Also it would be good to mention in commit message that this is a dummy (null) implementation

Reply:
Will do in next version.